be/src/exec/short_circuit.cpp

```diff
@@ -187,17 +187,15 @@ Status ShortCircuitExecutor::execute() {
     ChunkPtr chunk;
     bool eos = false;
     while (true) {
-        RETURN_IF_ERROR(_source->get_next(runtime_state(), &chunk, &eos));
```

Comment:
non-pipeline engine is deprecated and discarded,  we havn't to implement non-pipeline execution.